pdf-splitter-python/main_test.py

```diff
@@ -107,6 +108,98 @@ def test_split_document(self, mocked_client):
         # Clean up test files
         shutil.rmtree(temp_out_dir)
 
+    @patch("main.DocumentProcessorServiceClient")
+    def test_subdoc_with_3_plus_pages(self, mocked_client):
+        """Test for Document AI PDF splitter sample"""
+
+        # Create a mocked client instance to mock methods used by main.py
+        mocked_client_instance = MagicMock()
+
+        # Mock list processor API call to get a fake processor to use
+        mocked_client_instance.list_processors.return_value = docai.types.ListProcessorsResponse(
+            processors=[
+                docai.types.Processor(
```

Comment:
nit: I don't think you need to use `docai.types.Processor`

I've been able to just do `docai.Processor` (Same for all the types used below)